checks.d/couch.py

```diff
@@ -18,6 +18,7 @@ class CouchDb(AgentCheck):
     SERVICE_CHECK_NAME = 'couchdb.can_connect'
     SOURCE_TYPE_NAME = 'couchdb'
     TIMEOUT = 5
+    db_blacklist = []
```

Comment:
So, this won't play well in the (unlikely) case where you are monitoring multiple instances of couchdb on a single server, and if they have some database names in common.

If you want to blacklist one of these db in one of this instance, you'll end up blacklisting it for all instances.

So, the best (but kind of ugly) solution is to use a dictionary instead of a list whose keys are unique keys of an instance (e.g. the "server" parameter of the configuration).

Let me know if you have more questions